cdap-ui/package.json

```diff
@@ -146,95 +146,97 @@
   "dependencies": {
     "@apollo/react-hooks": "3.1.3",
     "@babel/polyfill": "7.0.0",
-    "@material-ui/core": "4.0.1",
-    "@material-ui/icons": "4.0.1",
-    "ace-builds": "1.4.4",
-    "apollo-boost": "0.4.4",
-    "apollo-server": "2.6.7",
-    "apollo-server-express": "2.6.7",
-    "body-parser": "1.14.2",
-    "bootstrap": "4.1.3",
+    "@material-ui/core": "4.8.2",
+    "@material-ui/icons": "4.5.1",
+    "ace-builds": "1.4.7",
+    "apollo-boost": "0.4.7",
+    "apollo-server": "2.9.15",
+    "apollo-server-express": "2.9.15",
+    "body-parser": "1.19.0",
+    "bootstrap": "4.4.1",
     "cask-datavoyager": "2.0.0-alpha.12.9",
     "cdap-avsc": "4.1.13",
-    "classnames": "2.2.5",
-    "clipboard": "1.7.1",
-    "compression": "1.6.1",
-    "cookie-parser": "1.4.1",
-    "css-vars-ponyfill": "1.8.0",
-    "d3": "4.12.2",
-    "dataloader": "1.4.0",
-    "ejs": "2.6.1",
-    "event-emitter": "0.3.4",
-    "express": "4.16.2",
-    "express-http-proxy": "1.5.1",
-    "finalhandler": "0.4.1",
-    "font-awesome": "4.6.3",
+    "classnames": "2.2.6",
+    "clipboard": "2.0.4",
+    "compression": "1.7.4",
+    "cookie-parser": "1.4.4",
+    "css-vars-ponyfill": "2.1.2",
+    "d3": "5.15.0",
+    "dataloader": "2.0.0",
+    "ejs": "3.0.1",
+    "event-emitter": "0.3.5",
+    "express": "4.17.1",
+    "express-http-proxy": "1.6.0",
+    "finalhandler": "1.1.2",
+    "font-awesome": "4.7.0",
     "frameguard": "3.1.0",
     "fuse.js": "3.4.6",
-    "graphql": "14.4.0",
+    "global": "^4.4.0",
```

Comment:
nit: I remember removing it somewhere but this is not a module that we use. Somewhere one of us must have done a `yarn add global <module-name>` instead of `yarn global add <module-name>` Can you remove this one?